packages/react-search-ui-views/src/__tests__/__snapshots__/ResultsPerPage.test.tsx.snap

```diff
@@ -9,17 +9,14 @@ exports[`renders correctly when there is a selected value 1`] = `
   >
     Show
   </div>
-  <StateManager
+  <ForwardRef
     className="sui-select sui-select--inline"
     classNamePrefix="sui-select"
     components={
       Object {
         "Option": [Function],
       }
     }
-    defaultInputValue=""
-    defaultMenuIsOpen={false}
-    defaultValue={null}
```

Comment:
We're not using these props and do not allow Search UI users to target them, so I think the diff is safe.